spec/features/services_spec.rb

```diff
@@ -32,6 +32,45 @@
       end
     end
 
+    describe 'viewing service list' do
+      context 'when there are more than 10 services' do
+        before do
+          14.times { |i| create_service(i) }
+          visit '/services'
+        end
+
+        it 'enables link to next page if service list is greater than 10' do
```

Comment:
the "... if service list is greater than 10" is redundant, as that's what the `context 'when there are more than 10 services'` sets up